fe1-web/src/features/wallet/components/__tests__/RollCallTokensDropDown.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { render } from '@testing-library/react-native';
+import React from 'react';
+
+import { mockKeyPair, mockLao, mockRC } from '__tests__/utils';
+
+import { PopToken } from '../../../../core/objects';
+import { RollCallToken } from '../../objects/RollCallToken';
+import { RollCallTokensDropDown } from '../index';
+
+const mockToken = PopToken.fromState(mockKeyPair.toState());
+const mockArray = [
+  new RollCallToken({
+    token: mockToken,
+    laoId: mockLao.id,
+    rollCallId: mockRC.id,
+    rollCallName: mockRC.name,
+  }),
+];
+
+describe('RollCallTokensDropDown', () => {
+  it('renders correctly', () => {
+    const { toJSON } = render(
+      <RollCallTokensDropDown
+        rollCallTokens={mockArray}
+        selectedTokenIndex={0}
+        onIndexChange={() => {}}
+      />,
+    );
+    expect(toJSON()).toMatchSnapshot();
+  });
```

Comment:
This is not important at all, but be aware that you can do this instead of the curly braces notation (which is commonly used when getting multiples elements).
```suggestion
  it('renders correctly', () => {
    const component = render(
      <RollCallTokensDropDown
        rollCallTokens={mockArray}
        selectedTokenIndex={0}
        onIndexChange={() => {}}
      />
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
```